charts/karpenter/templates/aggregate-clusterrole.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "karpenter.fullname" . }}-admin
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    {{- include "karpenter.labels" . | nindent 4 }}
+  {{- with .Values.additionalAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+rules:
+  - apiGroups: ["karpenter.sh"]
+    resources: ["provisioners", "provisioners/status"]
```

Comment:
should probably include the awsnodetemplate as well

Reply:
I've added the `awsnodetemplates` resource with read permissions. I'm not familiar with this CRD, and don't see it installed on a cluster I have running Karpenter. Is it only used when using custom launch templates?

Reply:
https://karpenter.sh/v0.13.2/aws/user-data/